### DISCUSSION TITLE
GitHub externship

### DISCUSSION THREAD

**Discussion by saurabhgithb** [github/india#618] [Students]
What requirements do we need to get short-listed in the GitHub externship?

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any `Official GitHub Externship Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Link to join: https://discord.gg/ceMXzhfaka

Thank you 🙏
